pkgs/development/python-modules/flax/default.nix

```diff
@@ -0,0 +1,60 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, keras
+, lib
+, matplotlib
+, msgpack
+, numpy
+, optax
+, pytestCheckHook
+, tensorflow
+}:
+
+buildPythonPackage rec {
+  pname = "flax";
+  version = "0.3.6";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0zvq0vl88hiwmss49bnm7gdmndr1dfza2bcs1fj88a9r7w9dmlsr";
+  };
+
+  propagatedBuildInputs = [
+    matplotlib
+    msgpack
+    numpy
+    optax
+  ];
```

Comment:
The `setup.py` file is mentioning [additional requirements](https://github.com/google/flax/blob/v0.3.6/setup.py#L27-L33).

Reply:
looks like it's only missing `jax`, perhaps because of a circular dependency issue?

Reply:
`jax` is already pulled in via `chex` -> `optax`, in fact it's redundant in their `setup.py` - not sure why they have it.